src/components/Common/SelectRecordListFields.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <b-button @click="showModal=true"
+      size="lg"
+      variant="light"
+    >
+      Select fields
+    </b-button>
+
+    <b-modal :visible="showModal"
+      @hide="onModalHide"
```

Comment:
Instead of @hide. Select fields on save.

Look at other modals that do that, I think its @ok